src/libstore/globals.hh

```diff
@@ -236,7 +236,7 @@ public:
         )",
         {"build-timeout"}};
 
-    PathSetting buildHook{this, true, "", "build-hook",
+    Setting<Strings> buildHook{this, {}, "build-hook",
```

Comment:
I don't think this is a problem, but this is a slight behaviour change as it means that we don't canonicalize the path any more. So having `build-hook = /doesnt-exist/../bin/nix` used to work but might not any more.